internal/reader/onchain_prices_reader_test.go

```diff
@@ -27,39 +29,26 @@ var (
 )
 
 func TestOnchainTokenPricesReader_GetTokenPricesUSD(t *testing.T) {
+	t.Skip("Skipping until we have a full price reader implementation")
```

Comment:
Should we unskip?

Reply:
In the next PR when we have BoundContracts

Reply:
Not sure I understand the reasoning behind that?